app/Http/Controllers/ActivitiesController.php

```diff
@@ -39,8 +39,11 @@ public function index(Request $request, Factory $factory): View
             ->with("selectedTag", $tag?->title);
     }
 
-    public function get(Factory $factory, string $slug): View
+    public function get(Request $request, Factory $factory, string $slug): View
     {
+        $urlPath = $request->getRequestUri();
+        $articleUrl = "https://blumilk.pl" . $urlPath;
```

Comment:
I don't get why URL is hardcoded here. We have `config("app.url")` value. But we can also add `/whatever` and browser should add main domain automatically.

Reply:
Added config here